source/features/index.tsx

```diff
@@ -82,7 +83,16 @@ let logError = (id: FeatureID, error: unknown): void => {
 
 // eslint-disable-next-line no-async-promise-executor -- Rule assumes we don't want to leave it pending
 const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
-	await elementReady('body', {waitForChildren: false});
+	while (!document.body) {
+		// eslint-disable-next-line no-await-in-loop
+		await Promise.race([delay(10), frame()]);
+	}
```

Comment:
If frame resolves, we definitely have body too.

```suggestion
	const oneFrame = frame();
	while (!document.body) {
		// eslint-disable-next-line no-await-in-loop
		await Promise.race([delay(10), oneFrame]);
	}
```

Maybe there's a "better" way to write it though


```suggestion
	await Promise.race([frame(), pWaitFor(() => document.body)]);
```

but unless we already have the dependency, I'd rather not

https://www.npmjs.com/package/p-wait-for

Reply:
> but unless we already have the dependency

its not